ios/Helio.xcodeproj/project.pbxproj

```diff
@@ -135,11 +131,13 @@
 		88BB34952829E9FB0057417B /* Montserrat-Bold.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 88BB34922829E9DC0057417B /* Montserrat-Bold.ttf */; };
 		88BB6015299BC09D003FA042 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 88BB6014299BC09D003FA042 /* GoogleService-Info.plist */; };
 		88BB6018299BC0B3003FA042 /* GoogleService-Info.plist in Resources */ = {isa = PBXBuildFile; fileRef = 88BB6016299BC0AF003FA042 /* GoogleService-Info.plist */; };
-		8DF077F7A73624EEB4761544 /* libPods-BetterSocial-tvOSTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = B880C1B4F2562553E5710094 /* libPods-BetterSocial-tvOSTests.a */; };
 		93C88B16B66646004C8065F1 /* libPods-OneSignalNotificationServiceExtension.a in Frameworks */ = {isa = PBXBuildFile; fileRef = BFF0B6565B1866252D7A81F7 /* libPods-OneSignalNotificationServiceExtension.a */; };
+		C36829349B058F4724332805 /* libPods-Helio-HelioDev.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 4D793740D13B5851280AE22A /* libPods-Helio-HelioDev.a */; };
+		CEB4A9AFC1C8EC4495C35A75 /* libPods-Helio.a in Frameworks */ = {isa = PBXBuildFile; fileRef = D3C2B9C5859E07404EC20481 /* libPods-Helio.a */; };
+		DE10BE8187E0741A2B59349E /* libPods-Helio-HelioStaging.a in Frameworks */ = {isa = PBXBuildFile; fileRef = AA5F02D50ECF85EDB43A2DBE /* libPods-Helio-HelioStaging.a */; };
 		E97154A42976870A0021A505 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 81AB9BB72411601600AC10FF /* LaunchScreen.storyboard */; };
 		E97154A52976870B0021A505 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 81AB9BB72411601600AC10FF /* LaunchScreen.storyboard */; };
-		FD367AE1785F0F10ECF31D7B /* (null) in Frameworks */ = {isa = PBXBuildFile; };
+		FDC358A01B89C324E9F57C41 /* libPods-Helio-tvOSTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 422C06FB4C3BA6EDC12E6514 /* libPods-Helio-tvOSTests.a */; };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this hunk seem to be related to the renaming of the app from "BetterSocial" to "Helio". The references to the old name have been replaced with the new one. However, it's important to ensure that these changes are reflected everywhere else in the codebase and that all dependencies are correctly updated to avoid any potential issues.
```diff
- 8DF077F7A73624EEB4761544 /* libPods-BetterSocial-tvOSTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = B880C1B4F2562553E5710094 /* libPods-BetterSocial-tvOSTests.a */; };
+ FDC358A01B89C324E9F57C41 /* libPods-Helio-tvOSTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 422C06FB4C3BA6EDC12E6514 /* libPods-Helio-tvOSTests.a */; };
```
Please make sure to test thoroughly after making such changes to ensure everything is working as expected.


<!-- This is an auto-generated comment by OSS CodeRabbit -->